src/hash.cr

```diff
@@ -1239,7 +1239,7 @@ class Hash(K, V)
     @size == 0
   end
 
-  # Calls the given block for each key-value pair and passes in the key and the value.
+  # Calls the given block for each key-value pair and passes in the key and the value as a `Tuple`.
```

Comment:
Add a type annotation for the block instead

Reply:
It will be `each(& : Tuple(K, V) ->) : Nil`.

Reply:
Good idea.

Reply:
OK check it now, thanks.

Reply:
The idea was to add that directly to the method signature, not in the documentation text.

Reply:
OK added it to the signature now, thanks!